### ISSUE TITLE
Add support for external addins.

### ISSUE THREAD

**Issue by Krzysztof-Cieslak** [spectresystems/jarvis#19]
It would be cool to support 3rd part addins. From quick scanning through the code it looks like it should be a simple addition - the list of assemblies with plugins is passed here - https://github.com/spectresystems/jarvis/blob/develop/src/Jarvis/Bootstrapper.cs#L40-L43 - and at the moment just is hardcoded to the existing plugins' assemblies. 

An easy fix would be to additionally load assemblies from `/addins` (or whatever, probably configurable) directory. More interesting would be adding some kind of "store" / repository for addins but it's probably overkill at the current stage.

**Comment by patriksvensson** [comment]
Yes. Addins are first class citizens in Jarvis, but I want to wait a little bit with opening it up until things are more stable and some other things are solved that I’m currently figuring out.
